lib/functions

```diff
@@ -118,6 +118,14 @@ function install_nodejs {
     npm install npm -g
 }
 
+# install_python() - Install Python 2.7 and other tools necessary for development.
+function install_python {
+    install_package python2.7
+    install_package python-dev
```

Comment:
Do you need the python-dev package? if you need it, you will need to add an assertion into the UT.

    # apt-cache show python-dev
    Package: python-dev
    ...
    Description-en: header files and a static library for Python (default)
     Header files, a static library and development tools for building
     Python modules, extending the Python interpreter or embedding Python
     in applications.
     .
     This package is a dependency package, which depends on Debian's default
     Python version (currently v2.7).